bpf/include/linux/byteorder.h

```diff
@@ -1,9 +1,7 @@
 #ifndef _ASM_X86_BYTEORDER_H
 #define _ASM_X86_BYTEORDER_H
 
-#include <endian.h>
-
-#if __BYTE_ORDER == __LITTLE_ENDIAN
+#if __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
```

Comment:
Do we need this header file at all any more?

Reply:
Doesn't seem to be needed anymore. Will remove it.

Reply:
Was a bit too quick to reply. Looks like it is still needed for `__constant_htonl` in `bpf/lib/utils.h`.